contrib/platform/src/com/sun/jna/platform/win32/Psapi.java

```diff
@@ -292,6 +294,18 @@ public interface Psapi extends StdCallLibrary {
      */
     boolean EnumProcesses(int[] lpidProcess, int cb, IntByReference lpcbNeeded);
 
+    /**
+     * Retrieves extended information about the pages at specific
+     * virtual addresses in the address space of the specified process.
+     *
+     * @param hProcess A Handle to the Process
+     * @param pv       A pointer to an array of PSAPI_WORKING_SET_EX_INFORMATION structures
+     * @param cb       The size of the pv buffer, in bytes.
+     * @return If the function succeeds, the return value is nonzero.
+     * @see <a href="https://docs.microsoft.com/en-us/windows/win32/api/psapi/nf-psapi-queryworkingsetex">MSDN</a>
+     */
+    boolean QueryWorkingSetEx(HANDLE hProcess, PSAPI_WORKING_SET_EX_INFORMATION pv, DWORD cb);
```

Comment:
'pv' should be a `Pointer` as the  user is passing an array.

Also to simplify mappings for users, use `int` for `cb` rather than `DWORD`.